src/account/PluginManagerInternals.sol

```diff
@@ -103,19 +102,28 @@ abstract contract PluginManagerInternals is IPluginManager {
         SelectorData storage _selectorData = getAccountStorage().selectorData[selector];
 
         if (!preExecHook.isEmpty()) {
```

Comment:
Now that we have `notEmpty()` shall we replace all instances of `!x.isEmpty()`?

Reply:
Yep, will do.